.pre-commit-config.yaml

```diff
@@ -19,13 +19,12 @@ repos:
   hooks:
   - id: isort
     args:
+    # Keep in sync with pyproject.toml
```

Comment:
Instead, move this to `.isort.cfg`. This is what I prefer normally: https://github.com/ansible/pylibssh/blob/devel/.isort.cfg.

Reply:
I saw both worked, I'm curious why the preference?

Reply:
In fact look at https://github.com/ansible-community/molecule/blob/main/pyproject.toml#L24-L26 -- which will also ensure you will not have surprises from black. No real need to add yet another dot config file in repository root. AFAIK there is no clear bug that would prevent us from using pyproject.toml

Reply:
@cidrblock explained at https://github.com/ansible/ansible-navigator/pull/777/files#r788706028. TL;DR it's hard to find where the configs are un a dump of sections from different tools that may also be present in other places — with dedicated files, it's easier for everybody to look up where the related settings live.

Reply:
Since both work fine and there doesn't seem to be a technical reason, leaving it `pyproject.toml` for now.  When/If something comes up where it makes sense to move it, it should not be a large effort.